packages/odyssey-react/src/components/Title/index.tsx

```diff
@@ -31,7 +38,18 @@ export type Props = {
   /**
    * The human readable section title to be visually displayed
    */
-  children: ReactText
+  children: ReactText,
+
+  /**
+   * Remove default block end margin
+   * @default false
+   */
+  noEndMargin?: boolean;
+
+  /*
+   * Specify explicit line height spacing
+   */
+  lineHeight?: 'base' | 'title';
```

Comment:
I've added these props as a temporary placeholder to keep the visual consistency intact. Once we have an underlying `View` / `Box` component composed within `Title` these can become much more generic and will pass their values through rather than applying classes that live within the `Title` CSS module.